src/socket-message-handling.ts

```diff
@@ -41,6 +121,8 @@ const setupWSConnection = (conn) => {
    */
   const subscribedTopics = new Set();
   let closed = false;
+  conn.binaryType = "arraybuffer";
```

Comment:
interesting, whats this for?

Reply:
This changes the binary type of received messages - since we are receiving ArrayBuffer and String data, we should have this line be split into two: `conn.binaryType = 'arrayBuffer'` and `conn.binaryType = 'blob'` where 'blob' is string data. These two lines should be moved into their respective resolvers as well.

Reply:
Still not sure what this is for. Why are we changing from the default? What does it do?

Reply:
The provider sends array buffer data (ex. <Buffer 01 0a 01 ae e0 e5 a9 0c 29 02 7b 7d>) to the receiver and the signaling service sends blob data (ex. {"type":"subscribe","topics":["new-room"]}) to the receiver. This line of code was included in the original y-websocket server.js file, so my thinking was that it is required to set the websocket binaryType to receive the array buffer data from the client. Is this not the case? I had trouble finding the answer to this online.

Reply:
Does it work if we take it out? If it's not necessary I would be happy to not add it.

Reply:
It works without it - I had it taken out for the most recent changes. Weird that it works without it (what does it even do...), and it makes the codebase more complicated with it in.